src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
 
+    public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+        int result = 0;
+        File file = new File(fromFileName);
+        StringBuilder builder = new StringBuilder();
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
+            String value = reader.readLine();
+            while (value != null) {
+                builder.append(value).append(" ");
+                value = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file", e);
+        }
+        String[] splitBuilder = builder.toString().split(" ");
+        for (int i = 0; i < splitBuilder.length; i++) {
+            String[] splitBuilderArray = splitBuilder[i].split(",");
+            if (splitBuilderArray[0].equals("supply")) {
```

Comment:
![image](https://user-images.githubusercontent.com/12864128/208706148-17689649-57ac-49f6-aecf-fa95a0c11a5f.png)
Use constant for indexes